miner.go

```diff
@@ -5,68 +5,81 @@ import (
 	"errors"
 	"io"
 
-	"github.com/filecoin-project/go-address"
-	"github.com/filecoin-project/specs-actors/actors/abi"
 	"github.com/ipfs/go-datastore"
 	logging "github.com/ipfs/go-log"
 	"golang.org/x/xerrors"
 
+	"github.com/filecoin-project/go-address"
+	sectorstorage "github.com/filecoin-project/sector-storage"
+	"github.com/filecoin-project/sector-storage/ffiwrapper"
+	"github.com/filecoin-project/specs-actors/actors/abi"
+	sealing2 "github.com/filecoin-project/storage-fsm"
```

Comment:
Is it still necessary to call this sealing2? Looks like sealing 1 is gone. Calling it fsm would also work.

Reply:
> Is it still necessary to call this sealing2? 

Nope. Good point. Will update.